### PR TITLE
Add Gui to the server

### PR DESCRIPTION
**Pull request by loucass003**
Combine both server and gui in the same repo.

I wanted to make gradle launch / compile both apps .... but could not find how to start two tasks in parallell
so i have not done it. If someone knows how it can be added later.

### CODE REVIEW

File: Gui/.github/workflows/build-gui.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Build GUI
```
Comment (TheDevMinerTV) [inline Gui/.github/workflows/build-gui.yml:1]: shouldn't be here

File: Gui/.husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```
Comment (TheDevMinerTV) [inline Gui/.husky/pre-commit:1]: shouldn't be here

File: Gui/.vscode/extensions.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (TheDevMinerTV) [inline Gui/.vscode/extensions.json:1]: won't get recognized by VSCode here

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

**Comment by Eirenliel** [comment]
so how will the history look like :nya_a:

also please use lower case letters for folder names...

**Comment by loucass003** [comment]
> so how will the history look like :nya_a:

History is gone

**Pull request closed without merging**